### ISSUE TITLE
[Unified Language Service] Text Analytics API Review

### ISSUE THREAD

**Issue by abhahn** [Azure/azure-rest-api-specs#16300]
For the first preview of the Language API including TA we plan to expose our existing async service, with a few minor alterations in order to align with internally discussed naming conventions.

Swagger PR for first preview version:  https://github.com/Azure/azure-rest-api-specs/pull/16256

**Comment by markweitzel** [comment]
# API Stewardship Review 12-Oct-21

Today's conversation will be focused on the batch API described in the design doc.
Internal design doc:  [Language Unified REST API](https://microsoft.sharepoint-df.com/:w:/t/ALPSAzureLanguagePillars/EZPRUkaOsN1HscCEdBIvFooBsIlxftyxrfzPhHTtVko_vw?e=IqzzpE)

* We are switching this to `records` from `document` b/c documents has a broader definition w/in Cognitive. `records` has a broader meaning. 
* There will be existing APIs that overlap with the names. These will adhere to the breaking change rules. 
  * All of the inputs & outputs should be the same.
  * There will be impact to the SDK team when building the client libraries.
*

**Comment by abhahn** [comment]
Hi @markweitzel , the PR for this API version has been updated according to prior feedback, and also after a long discussion over email regarding the optional `records` property (which has been reverted back to `documents`).  I can share more details about the email thread if necessary, but we have decided to just make all of the mutually exclusive options within the analysisInput object optional instead of required for simplicity and to avoid the need to complicate the structure.  This should avoid the breaking change later when / if we add additional properties to this object in the future.
